lib/src/delegates/asset_picker_builder_delegate.dart

```diff
@@ -1553,12 +1590,11 @@ class DefaultAssetPickerBuilderDelegate
   int findChildIndexBuilder({
     required String id,
     required List<AssetEntity> assets,
+    required List<SpecialItemResult> prependSpecialItemResults,
```

Comment:
Same as above

Reply:
done updating function argument to specialItemModels.